server/conf/messages.am

```diff
@@ -436,3 +436,16 @@ adminValidation.multiOptionEmpty=ባለብዙ አማራጭ ጥያቄዎች ባ
 error.notFoundTitle=ለመጎብኘት የሞከሩትን ገጽ ማግኘት አልቻልንም
 error.notFoundDescription=«እባክዎ ወደኋላ ይሂዱ ወይም ይመለሱ ወደ »
 error.notFoundDescriptionLink=መነሻ ገጽ
+
+#-------------------------------------------------------------#
+# EMAILS - boilerplate text contained in emails sent to users #
+#-------------------------------------------------------------#
+email.applicationReceivedBody=የ{0} ፕሮግራም ማመልከቻዎ ደርሷል። የአመልካች መታወቂያዎ {1} ነው፣ እና የማመልከቻ መታወቂያው {2} ነው
+email.applicationReceivedSubject=የ{0} ፕሮግራም ማመልከቻዎ ደርሷል
+email.applicationUpdateSubject=An update on your application {0}
```

Comment:
That makes sense.  We can hang onto this PR until we get the next batch of translations, or I could split out the i18n stuff into a separate PR.  That kind of scope creeped into this PR.

Reply:
Sounds good! Unfortunately it can take up to 2 weeks for new strings to come back as a heads up, but if that's okay, keeping this until those come back sound fine to me!

Reply:
I'll check with Isabel.  I think there's no rush to get this in, but if we want to just close it out, I'll remove these.

Reply:
I just took those three out of the non-English translations, so I think we can merge this whenever.